primary/views/manage/index.pug

```diff
@@ -66,10 +69,10 @@ block content
 		//- 		span Track/simulate alliance selection
 		a(href="manage/currentevent/getcurrentteams") 
 			div(class="gear-btn theme-link w3-btn")
-				span Update list of current teams
+				span!=msg('layout.nav.manage.event.getcurrentteams')
```

Comment:
`layout.nav` was kinda meant to be a temporary place for strings like these, it would be better to have them under where they actually are (`manage`)